source/Government.cpp

```diff
@@ -164,6 +164,16 @@ void Government::Load(const DataNode &node)
 		{
 			if(key == "provoked on scan")
 				provokedOnScan = false;
+			else if(key == "reputation")
+			{
+				for(const DataNode &grand : child)
+				{
+					if(grand.Token(0) == "max" && grand.Size() >= 2)
+						reputationMax = numeric_limits<double>::max();
+					else if(grand.Token(0) == "mint" && grand.Size() >= 2)
```

Comment:
Mint. :flushed:

Reply:
These rep limits are mint, you don't even know.